react/features/base/config/reducer.ts

```diff
@@ -163,6 +169,8 @@ function _setConfig(state: IConfig, { config }: { config: IConfig; }) {
     // eslint-disable-next-line no-param-reassign
     config = _translateLegacyConfig(config);
 
+    _setDeeplinkingDefaults(config.deeplinking as IDeeplinkingConfig);
```

Comment:
Doesn't this kinda belong inside `_translateLegacyConfig` ?

Reply:
I moved it there. Though my thought was that this is not really related to legacy, as we only set some defaults not related to old config values.

Reply:
I think separating it to another function looks good! But we should probably call it inside there.

I don't see a future where we completely get rid of it though :-)